dvc/command/data_sync.py

```diff
@@ -116,7 +116,10 @@ def shared_parent_parser():
         "targets",
         nargs="*",
         default=None,
-        help="Limit the scope to these stage files.",
+        help=(
+            "Limit the scope to these DVC-files."
+            " With -R a directory to search DVC-files in can be specified."
```

Comment:
This affects documentation changes which weren't sent to the dvc.org repo. I'll fix it, but just a note. Thanks